### PR TITLE
14/connect front end back end

### PR DESCRIPTION
**Pull request by davidbenicek**

### CODE REVIEW

File: back_end/RayTracer/Views/Index/Index.aspx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿<%@ Page Language="C#" Inherits="System.Web.Mvc.ViewPage" %>
```
Comment (hiteshmankani95) [inline back_end/RayTracer/Views/Index/Index.aspx]: This seems like a testing file?

**Review by hiteshmankani95 — COMMENTED** [review]
Everything looks good! Just a test file to remove

File: back_end/RayTracer/Views/Index/Index.aspx (hunk 1 above)
Comment (davidbenicek) [reply back_end/RayTracer/Views/Index/Index.aspx]: Yeh you're right. Just got rid of it! Thanks :)

File: back_end/RayTracer.userprefs [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+      <File FileName="RayTracer/Controllers/IndexController.cs" Line="38" Column="9" />
+      <File FileName="RayTracer/App_Start/RouteConfig.cs" Line="1" Column="1" />
+      <File FileName="RayTracer/packages.config" Line="3" Column="32" />
+      <File FileName="RayTracer/Views/Web.config" Line="4" Column="82" />
     </Files>
   </MonoDevelop.Ide.Workbench>
   <MonoDevelop.Ide.Workspace ActiveConfiguration="Debug" />
   <MonoDevelop.Ide.DebuggingService.Breakpoints>
     <BreakpointStore>
-      <Breakpoint file="/Users/othmanalkhamra/Documents/GitHub/raytracer/back_end/RayTracer/Models/Json/JsonObject.cs" relfile="RayTracer/Models/Json/JsonObject.cs" line="32" column="1" />
```
Comment (hiteshmankani95) [inline back_end/RayTracer.userprefs]: Just noticed this, probably shouldn't have that in git

Comment (davidbenicek) [reply back_end/RayTracer.userprefs]: Deleted :)

**Review by hiteshmankani95 — APPROVED** [review]

**Pull request merged**